services/QuillLMS/app/serializers/diagnostic_insights/student_learning_sequence_activity_serializer.rb

```diff
@@ -19,15 +19,19 @@ def self.recommendations(diagnostic_ids)
     PRE_RECOMMENDATION = 'pre_recommendation'
     POST_RECOMMENDATION = 'post_recommendation'
 
-    attributes :activity_id, :activity_type, :completed_at, :created_at, :activity_session_id, :proficient, :percentage, :timespent
+    # We've decided that if a student is proficient in 80% of the skills on a diagnostic, they're considered "proficient", this is subject to change based on feedback and testing with students and teachers
+    SKILLS_MASTERED_PROFICIENCY_THRESHOLD = 0.80
+
+    attributes :activity_id, :activity_type, :completed_at, :created_at, :activity_session_id, :proficient, :percentage, :skill_performance, :timespent
 
     def percentage = object.activity_session_unscoped&.percentage
     def timespent = object.activity_session_unscoped&.timespent
 
     def proficient
-      return nil unless pre_diagnostic? || post_diagnostic?
+      return unless pre_diagnostic? || post_diagnostic?
```

Comment:
You might consider avoiding combining an `unless` with an `or` here. Consider making a variable `def diagnostic? = pre_diagnostic? || post_diagnostic?` so this could be `return unless diagnostic?`